src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -44,18 +44,16 @@
     //}
     // See https://docs.orchardcore.net/en/latest/docs/reference/modules/Media.AmazonS3/#configuration to configure media storage in Amazon S3 Storage.
     //"OrchardCore_Media_AmazonS3": {
-    //  "BucketName": "somebucketname",
-    // Credentials section needed only if Orchard will be hosted not in the AWS Cloud
-    // since in AWS you don't need to store this info in the appsettings file, you just
-    // need to set BucketName and BasePath.
-    //    "Credentials": {
-    //      "SecretKey": "",
-    //      "AccessKeyId": "",
-    //      "RegionEndpoint": "eu-central-1"
-    //    },
-    //    "BasePath": "/media",
-    //    "ProfileName": "",
-    //    "CreateBucket" : true
+    //  "Region": "eu-central-1",
+    //  "Profile": "defualt",
+    //  "ProfilesLocation": "",
+    //  "Credentials": {
+    //    "SecretKey": "",
+    //    "AccessKey": ""
+    //  },
+    //  "BasePath": "/media",
+    //  "CreateBucket": false,
```

Comment:
Any specific reason you flipped this?

Reply:
Actually no, I've just wanted to have a distinction between Aws specific configuration params and Orchard specific params (like prefix, CreateBucket flag etc.)

Reply:
I think it would be useful to keep it as true.